### DISCUSSION TITLE
V3: no clickable rows when using Page

### DISCUSSION THREAD

**Discussion by donnyxray** [filamentphp/filament#7155] [General]
I've noticed that clickable rows work fine when I use the default pages:

```php
class Something extends ListRecords
{
    public function table(Table $table): Table
    {
    //...
    }
}
```

However, when I use a custom page, the rows are not clickable:

```php
class Something extends Page implements HasTable
{
    use InteractsWithTable;

    public function table(Table $table): Table
    {
    //...
    }
}
``` 

Is this expected? or is some additional code needed to make the rows clickable on custom pages?

**Comment by danharrin** [comment]
https://beta.filamentphp.com/docs/3.x/tables/advanced#record-urls-clickable-rows

**Comment by donnyxray** [comment]
Thank you!
